examples/container/V2containers/DeleteMonitoringV2/main.go

```diff
@@ -36,8 +36,8 @@ func main() {
 	monitoringAPI := monitoringClient.Monitoring()
 
 	var monitoringInfo = v2.MonitoringDeleteRequest{
-		Cluster:  "DragonBoat-cluster",
-		Instance: "ec4f0886-edc4-409e-8720-574035538f91",
```

Comment:
Don't hard code the values..may be ask them as parameters which end user can pass via command line..refer old examples
```
ar org string
	flag.StringVar(&org, "org", "", "Bluemix Organization")

	var space string
	flag.StringVar(&space, "space", "", "Bluemix Space")

	var skipDeletion bool
	flag.BoolVar(&skipDeletion, "no-delete", true, "If provided will delete the resources created")

	flag.Parse()

	if org == "" || space == "" {
		flag.Usage()
		os.Exit(1)
	}
	```